packages/broccoli-eyeglass/src/types/sync-disk-cache.d.ts

```diff
@@ -1,16 +1,32 @@
 export = SyncDiskCache;
+interface CacheHit {
+  isCached: true;
+  key: string;
+  value: string;
+}
+interface CacheMiss {
+  isCached: false;
+  key: undefined;
+  value: undefined;
+}
+
+interface SyncDiskCacheOptions {
+  location?: string;
+  compression?: 'deflate' | 'deflateRaw' | 'gzip';
+}
+
 declare class SyncDiskCache {
```

Comment:
left and issue, to simply port sync-disk-cache to typescript so that it brings it's own typings with: https://github.com/stefanpenner/sync-disk-cache/issues/17 (if you have free time, I would welcome the PR)

Reply:
you don't have to port to typescript to provide typings. you can just publish this type definition with your project and keep it up to date when the public API changes.

Reply:
Yup you are correct, but I have been finding the TS port sorta cathartic :P Good evening while watching TV exercise.  Plus I can write fancy ES2018 syntax, for old nodes.